nbval/plugin.py

```diff
@@ -16,13 +16,7 @@
 import warnings
 from collections import OrderedDict, defaultdict
 
-# for python 3 compatibility
-import six
-
-try:
-    from Queue import Empty
-except:
-    from queue import Empty
+from Queue import Empty
```

Comment:
```suggestion
from queue import Empty
```

The lowercase name is the Python 3 variant.

Reply:
updated. thx